R/looptools.R

```diff
@@ -0,0 +1,106 @@
+#' @title compute two-point correlation function between quark loops
+#' @param loop_snk 'raw_cf' container with spin-projected quark loop
+#'                 at sink (see \link{loop_spin_project})
+#' @param loop_src 'raw_cf' container with spin-projected quark loop 
+#'                 at source (see \link{loop_spin_project})
+#' @return 'raw_cf' container with two-point function of these two quark loops.
+#'         In the calculation, both averaging over all source locations and
+#'         the average over all stochastic sample combinations are performed.
+#' @export
+loop_2pt <- function(loop_snk, loop_src){
```

Comment:
does this replace `computeDisc`?

Reply:
Not necessarily. I just needed a function to handle the 'raw_cf' format to work comfortably with the data that I have since complex multiplications are not overloaded for 'cf'. 'raw_cf' kind of formalises how to deal with data that can generally be complex and has more than measurement and time dimensions. For the loops, for instance, one can have a "samples" dimension as well as two Dirac dimensions.

Reply:
Since the input arguments for ```computeDisc``` are not actually real ```cf``` objects, it's also not quite straightforward to drop down from ```raw_cf``` to ```cf``` and then use ```computeDisc```. However, I think in practice one would load data into ```raw_cf``` for first analysis to see if everything looks kosher and then construct the necessary input for ```computeDisc``` from that. I think a renaming of ```computDisc``` might be good to clarify that his actually computes a two-point correlation.